### ISSUE TITLE
VGGT-Omega feature head: the highest-headroom backbone, six sequences

### ISSUE THREAD

**Issue by fukka** [fukka/vggt-omega#33]
# VGGT-Omega feature head: the highest-headroom backbone, six sequences

**Owner:** gpu
**Status:** open — not started.
**Files I may touch:** none — runs only (scratch runners fine, as in #29).
Results to `results` under `results/autoresearch-h24-omega/`.
**Blocked by:** none. Code on `organized`:
`autoresearch/experiments/h2-center-safe-adapter/code/omega_head.py`
(protocol `protocol-h2.4-vggt-omega.md` locked; full path exercised CPU-side
with `--random-init` — structure only).
**What is waiting on it:** VGGT-Omega has the largest distance-controlled rim
penalty of the five ticket-024 models (1.81×) and is this repo's own
backbone. DA3-Small's head held on all six sequences (#29). If the same
recipe works on VGGT-Omega, the backbone-agnostic claim is complete; if not,
the failure localizes what its final tokens lack.

## The task

For each of the six #29 sequences, both splits:

```bash
python autoresearch/experiments/h2-center-safe-adapter/code/omega_head.py \
    --seq <seq_dir> --split halves --weights <the VGGT-Omega-1B-512 checkpoint
    fovbench's vggt_omega backend loads> --out results/run_013_<seq>_halves.json
# and --split even_odd
```

Notes:
- `VGGTOmega()` is instantiated at default width (1024). If
  `load_state_dict(strict=True)` fails on the fovbench checkpoint, load with
  strict=False and put the missing/unexpected key lists in the comment —
  aggregator + dense_head must load fully for the run to count.
- Depth is converted planar-z → range through the calibrated KB4 camera inside
  the script (convention discipline; don't re-convert).
- Same *.png-glob caveat as #29 if it applies; scratch-runner it.
- GPU makes the 1B forwards cheap; caches land under $H2_CACHE/<seq>/.

## Acceptance

- 12 JSONs on `results` + a comment: per-sequence near-rim before→after next
  to #29's DA3-Small numbers, and one sentence: does the recipe transfer to
  the highest-headroom backbone?

**Comment by fukka** [comment]
Ticket [031](https://github.com/fukka/vggt-omega/blob/organized/docs/handoff/tickets/031-omega-head-six-sequences.md). Pushed to `results@6828118`, `results/autoresearch-h24-omega/` (12 JSONs). `omega_head.py` run unmodified with the real checkpoint (`checkpoints/VGGT-Omega-1B-512/model.pt`, the path `finetune/eval/baselines/model_zoo.py`'s `OMEGA_CKPT` default resolves to) — `load_state_dict(strict=True)` succeeded, no fallback needed, feat dim 2048 confirms real weights.

Near-rim before→after, all six sequences, both splits, next to DA3-Small's #29 numbers:

| sequence | split | VGGT-Omega | DA3-Small (#29) |
|---|---|---|---|
| seq131 | halves | 0.157→0.127 (-19.3%) | -46.7% |
| seq131 | even_odd | 0.185→0.110 (**-40.6%**) | -67.4% |
| seq133 | halves | 0.201→0.137 (-31.9%) | -66.8% |
| seq133 | even_odd | 0.193→0.140 (-27.4%) | -72.7% |
| seq134 | halves | 0.193→0.155 (-19.7%) | -64.6% |
| seq134 | even_odd | 0.182→0.131 (-27.9%) | -69.3% |
| seq135 | halves | 0.199→0.148 (-25.6%) | -71.3% |
| seq135 | even_odd | 0.214→0.136 (**-36.1%**) | -75.2% |
| seq136 | halves | 0.250→0.201 (-19.7%) | -69.9% |
| seq136 | even_odd | 0.200→0.138 (-31.3%) | -71.3% |
| decoration_seq132 | halves | 0.118→0.082 (-30.5%) | -21.2% |
| decoration_seq132 | even_odd | 0.144→0.089 (**-38.1%**) | -32.7% |

**Does the recipe transfer to the highest-headroom backbone? Yes, but with a smaller effect size and more collateral.** Near-rim AbsRel improves on all 12 runs (-19.3% to -40.6%) — the backbone-agnostic claim holds — but VGGT-Omega never reaches DA3-Small's -60/-70%+ range on seq133–136. It also costs more: near_center regresses by up to **+49.8%** (seq134 halves) and +44–45% on several other halves splits, well past DA3-Small's <+6% collateral (even_odd splits stay closer to DA3-Small's range, -4.8% to +12.3%). Read: VGGT-Omega's final-block tokens carry less of whatever signal lets DA3-Small's head fix the rim without touching the center — same recipe, weaker and messier on this backbone.

Same `*.png`/`*.jpg` glob workaround as #29/#32 (scratch symlink tree, not committed).

**Comment by fukka** [comment]
Consumed CPU-side: backbone-agnostic directionally (12/12 improve) but weaker and with halves-split center collateral up to +50% — recorded as supported-with-caveats; earlier-block probing filed as future work, not a new ticket now. Closing.
